docs/conda_windows_install.md

```diff
@@ -0,0 +1,18 @@
+# Installing popsborder with conda on Windows
+
+As an alternative to pipenv, you can manage the Python packages needed to run popsborder using a conda environment.
+
+Unless you plan to use additional features of the Anaconda suite (e.g. Anaconda Navigator, Jupyter Notebook, and other tools), we recommend installing Miniconda, which contains only the conda package manager and Python.
+
+Follow the instruction to download and install Miniconda: https://docs.conda.io/en/latest/miniconda.html
+
+Once installed, open Command Prompt and run the following lines, one at a time:
+
+```
```

Comment:
The Super-Linter wants to know the language of the code block here.  I'm not sure what is appropriate for Windows command line. Maybe `bat` is okay here (we render it on GitHub, so their [Batchfile](https://github.com/github/linguist/blob/73e2d735c3c26577fc89c1cb3f8342e5a8ff1d82/lib/linguist/languages.yml#L484) category should apply).

The syntax is:

````
```bat
conda...
```
````

```
 /github/workspace/docs/conda_windows_install.md:11 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]]
```